awesomemotive/affiliatewp-leaderboard
# Issue/34 - Added shortcode atts sanitization

Issue #34 

This is just to ensure that the shortcode arguments are escaped properly before going ahead.
The test is just to certify if the shortcode runs properly after these addition.

**Steps to test:**

- Install Leaderboard addon;
- In WP Admin, go to Pages and use the shortcode ` [affiliate_leaderboard number="5" referrals ="yes" visits ="yes" earnings ="yes" orderby="referrals"]`, try to change the parameters and check if is working accordingly to the [docs](https://affiliatewp.com/docs/leaderboard-installation-and-usage/);
- You can also try using some JS injection like this [affiliate_leaderboard number="'); alert(1); ('" status="'); alert(1); ('"], it should not display any JS alerts.


**Housekeeping**


**Proposed changes**
